wd_helper.au3

```diff
@@ -1254,9 +1254,14 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 	Local $iErr = $_WD_ERROR_Success, $sDriverEXE, $sBrowserVersion, $bResult = False
 	Local $sDriverCurrent, $sVersionShort, $sDriverLatest, $sURLNewDriver
 	Local $sTempFile, $oShell, $FilesInZip, $sResult, $iStartPos, $iConversion
+	Local $bCheckArch = False
```

Comment:
In general good concept.
But
`$bCheckArch` <> `$bKeepArch`

Reply:
Fixed